components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -70,11 +68,50 @@ namespace brave {
 
 namespace {
 
+// NOTE TO READER: This header is for partners to determine the browser is
+// Brave without creating webcompat issues by creating a new user-agent.
+// All Brave users send the exact same header and so this cannot be used for
+// tracking individual users. See
+// https://github.com/brave/brave-browser/wiki/Custom-Headers for more
+// information. Custom headers are deprecated and new partners use the
+// navigator.brave.isBrave() JavaScript API.
+constexpr char kPartnerEaffName[] = "eaff";
+constexpr char kPartnerUpholdName[] = "uphold";
+constexpr char kPartnerGrammarlyName[] = "grammarly";
+
+const std::vector<std::string> kPartnerEaffDomains = {"eaff.com",
```

Comment:
these are prohibited by the style guide, see https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

pls change to `std::array` or plain C arrays

Reply:
fixed